go/memcached-operator/deploy/olm-catalog/memcached-operator/memcached-operator.package.yaml

```diff
@@ -1,5 +1,6 @@
 channels:
-- currentCSV: memcached-operator.v0.12.0
-  name: alpha
-defaultChannel: alpha
-packageName: memcached-operator
+    - currentCSV: memcached-operator.v0.13.0
+      name: alpha
+    defaultChannel: alpha
+    packageName: memcached-operator
+    
```

Comment:
Shows that the indentation here is not accurate.

Reply:
Nit: revert the extraneous whitespace changes at the end of the file

Reply:
Fixed